pkg/http_endpoint/v1/handlers/http_grafana.go

```
@@ -0,0 +1,337 @@
+						http.StatusText(http.StatusInternalServerError),
+						"provide Either Name or Email in parameters")
+					return
+				}
+			} else {
+				if !(emailValid.MatchString(res.Email)) {
+					common.WriteErrorToResponse(w, http.StatusInternalServerError,
+						http.StatusText(http.StatusInternalServerError),
+						"Email Invalid")
+					return
+				}
+			}
+			if len(res.Login) == 0 {
+				common.WriteErrorToResponse(w, http.StatusInternalServerError,
+					http.StatusText(http.StatusInternalServerError),
+					"provide Login in parameters")
+				return
+			}
+			if len(res.Password) == 0 {
+				common.WriteErrorToResponse(w, http.StatusInternalServerError,
+					http.StatusText(http.StatusInternalServerError),
+					"provide Password in parameters")
+				return
+			}
+
+			// Create user if no error
+			handler.CreateUser(clients, res, w)
+		}
+	}
+}
+
+// GetOrganization method gets the list of organizations
+func GetOrganization(clients *common.ClientContainer, handler common.HandlerInterface) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		orglist, err := handler.GetOrganizations(clients, w)
+		if err != nil {
+			log.Logger.Error(err)
+			return
+		}
+		orgs, _ := json.Marshal(orglist)
+		w.WriteHeader(http.StatusOK)
+		w.Write(orgs)
+	}
+}
+
+// GetOrganizationID method gets the organization by ID
+func GetOrganizationID(clients *common.ClientContainer, handler common.HandlerInterface) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		organizationID := chi.URLParam(r, "organizationID")
+		ID, err := strconv.Atoi(organizationID)
+		if err != nil {
+			log.Logger.Error(err)
+			common.WriteErrorToResponse(w, http.StatusInternalServerError,
+				http.StatusText(http.StatusInternalServerError),
+				"provided organizationID is not integer")
+			return
+		}
+		orglist, err := handler.GetOrganizationID(clients, w, ID)
+		if err != nil {
+		}
+		orgs, _ := json.Marshal(orglist)
```

serialization error is not handled here